### PR TITLE
sc2: Removing unused dependency in requirements.txt

### PR DESCRIPTION
**Pull request by MatthewMarinets**
## What is this fixing or adding?
Removing an unused `six` dependency identified in #ap-core-dev a few minutes ago. Seems it's just been there unused since time immemorial.

## How was this tested?
Did a `pip uninstall six`, and:
* Ran unit tests
* Generated
* Started the client and connected to a locally-hosted server
* Started a mission

## If this makes graphical changes, please attach screenshots.
None

### CODE REVIEW

**Comment by MatthewMarinets** [comment]
Pinging @Ziktofel to request review

**Comment by black-sliver** [comment]
Can you fix the missing newline at end of the file while you are at it, so next time it won't have to change an extra line?

File: worlds/sc2/requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-nest-asyncio >= 1.5.5
-six >= 1.16.0
\ No newline at end of file
+nest-asyncio >= 1.5.5
```
Comment (Exempt-Medic) [inline worlds/sc2/requirements.txt:1]: EoF newline
```suggestion
nest-asyncio >= 1.5.5

```

**Review by Exempt-Medic — APPROVED** [review]
LGTM. They said it was tested so it should be fine

**Review by Ziktofel — APPROVED** [review]

**Review by black-sliver — APPROVED** [review]

**Pull request merged**